### PR TITLE
[NOTEST] [RFR] EC2cleanup boto3 refactor from object to dict

### PR DESCRIPTION
**Pull request by mmojzis**
Wrapanapi ec2 is moving to boto3 and in boto3 response returns dict instead of object so I needed to change calling of elb.name.
Depends on this PR:
https://github.com/ManageIQ/wrapanapi/pull/362

{{ pytest: cfme/tests/cloud* --use-provider ec2west --long-running }}

### CODE REVIEW

**Comment by digitronik** [comment]
@mmojzis for now moving to `WIPTEST`. as `wrapanapi` PR merge please move this to `RFR`.

**Comment by mshriver** [comment]
@mmojzis Please update the requirements files to point to your wrapanapi branch. This will allow PRT to actually run with your wrapanapi changes directly.  It will help us identify the dependency on wrapanapi as well.

When wrapanapi gets released, we can update your requirements changes to point to a specific wrapanapi version.

**Comment by mmojzis** [comment]
@mshriver Added reference to my wrapanapi branch.

**Comment by dajoRH** [comment]
Lint report for commit b9f2736c9fd3ff0999b526d328170786ebcb99f4:

`scripts/ec2cleanup.py`:
- :bangbang: Line 143:47: **E231** *missing whitespace after ','*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mmojzis** [comment]
@mshriver Please after merge can you trigger jenkins job manually so we can truly verify the functionality? I tested it for myself and it works well, but who knows how it behaves for jenkins.

File: scripts/ec2cleanup.py [hunk 1]
```diff
@@ -81,11 +82,14 @@ def delete_unattached_volumes(provider_mgmt, excluded_volumes, output):
     try:
         for volume in provider_mgmt.get_all_unattached_volumes():
             if excluded_volumes and volume.id in excluded_volumes:
-                logger.info("  Excluding volume id: %r", volume.id)
+                logger.info("  Excluding volume id: %r", volume["VolumeId"])
                 continue
             else:
-                volume_list.append([provider_name, volume.id])
-                volume.delete()
+                if datetime.now(pytz.utc) - volume["CreateTime"] > timedelta(hours=3):
```
Comment (digitronik) [inline scripts/ec2cleanup.py]: It it not possible to use `datetime.utcnow()`? really we are facing `daylight savings` problem in this manipulation?

Comment (mmojzis) [reply scripts/ec2cleanup.py]: I had it there, but AWS output is always offset-aware and then I get Exception:
`TypeError: can't subtract offset-naive and offset-aware datetimes`

**Pull request merged**